frontend/src/app/features/invite-user-modal/success/success.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="op-ium-success">
-  <div class="op-modal--body op-modal--body_center">
+  <div class="spot-modal--body spot-modal--body_center">
```

Comment:
The `_center` class should be a special override for just this component for now